docker/redis/redis.conf

```diff
@@ -0,0 +1,1812 @@
+# Redis configuration file example.
```

Comment:
Is that file required ? I don't recall we ever used it on projects till now

Reply:
not needed, should be removed in favour of `command` addition

Reply:
Removed commented and empty lines in docker/redis/redis.conf to make it clearer what to move